src/pages/OPP_SendAcknowledgmentBTN.page

```diff
@@ -2,26 +2,21 @@
     extensions="OPP_SendAcknowledgmentBTN_CTRL"
     recordSetVar="listOpp"
     action="{!SendAcknowledgment}"
-    standardStylesheets="false"
-    title="{!$Label.sendAcknowledgmentPageTitle}">
-    <apex:slds />
+    standardStylesheets="true"
+    title="{!$Label.sendAcknowledgmentPageTitle}"
+    docType="html-5.0">
 
+    <apex:slds />
+    
     <apex:form id="vfForm">
         <div class="slds">
             <!-- PAGE HEADER -->
-            <div class="slds-page-header slds-p-bottom_small" role="banner">
-                <div class="slds-grid">
-                    <div class="slds-col">
-                        <h1 class="slds-text-heading_medium">{!$Label.sendAcknowledgmentPageTitle}</h1>
-                    </div>
-                    <div class="slds-col slds-no-flex slds-align-middle">
-                        <div class="slds-button-group" role="group">
-                            <apex:commandButton id="close" styleClass="slds-button slds-button_neutral" action="{!close}"
-                                value="{!$Label.sendAcknowledgmentReturnLinkLabel}" immediate="true" reRender="vfForm"/>
-                        </div>
-                    </div>
-                </div>
-            </div>
+            <c:UTIL_PageHeader headerLabel="{!$ObjectType.Opportunity.Label}"
+                               header="{!$Label.sendAcknowledgmentPageTitle}"
+                               icon="opportunity_120" iconCategory="standard"
+                               showSaveBtn="false" cancelLabel="{!$Label.bdiBtnClose}"
```

Comment:
Should the Cancel label be `$Label.sendAcknowledgmentReturnLinkLabel`?

Reply:
I believe that this button can be called either from a specific Opportunity or from a list of Opportunities. In the case of the latter, it would make more sense to have the button label of Close instead of "Return to Opportunity".  Is that correct @jjbennett?

Reply:
@RadSFDeveloper @force2b Yes, that is correct so I changed it to "Close" to work in either case.